roles/tftp-server/files/late_command.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+set -euf
+
+# This script setups ansible and runs it
+# It should be ran at the end of the basic installation of a machine
+
+apt install -y ansible git eatmydata
+
+# We clone our ansible repository and copy the ansible config files
+
+git clone https://anonscm.debian.org/git/debconf-video/ansible.git /root/debconf-ansible
+git clone https://github.com/xfxf/lca2017-av.git /root/lca2017-av
+
+ln -s /root/lca2017-av/inventory/ansible-up.sh /usr/local/sbin/ansible-up
```

Comment:
It lets you re-run without your laptop, and all the ansible stuff checked out. I found it useful during the sprint. But didn't use it during the conference.